playbooks/sync-task.yml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Configure instance(s)
+  hosts: all
+  become: True
+  gather_facts: No
+  tasks:
+    - name: Add script for syncing logs on exit
+      template:
+        src: sync-logs-on-exit.j2
+        dest: "{{ COMMON_BIN_DIR }}/sync-logs-on-exit"
+        owner: root
+        group: root
+        mode: 0744
+      when: COMMON_OBJECT_STORE_LOG_SYNC
```

Comment:
@Metfriet, you can remove the `when` condition because we always want these tasks to run - we have defined the variable in the PRs already merged.